sample-script/watchResource.js

```diff
@@ -0,0 +1,87 @@
+const { BotkitConversation } = require("botkit");
+
+module.exports = function (controller) {
+    let stop;
+
+    controller.hears(new RegExp(/((watch)$)|((watch) [a-zA-Z]*$)/), "direct_mention", async (bot, message) => {
+        var trimmedMessage = message.text.replace("watch ", "");
+        if (stop != undefined) {
+            console.log("Watch has already been started!");
+
+            await bot.reply(message, {
+                blocks: [
+                    {
+                        type: "section",
+                        text: {
+                            type: "mrkdwn",
+                            text: "Uh oh! You already have started a watch!",
+                        },
+                    },
+                ],
+            });
+            return;
+        }
+
+        await bot.reply(message, {
+            blocks: [
+                {
+                    type: "section",
+                    text: {
+                        type: "mrkdwn",
+                        text: `Kubernetes watch running on ${trimmedMessage}!`,
+                    },
+                },
+            ],
+        });
+
+        const kubeResource = {
+            type: trimmedMessage,
```

Comment:
We should probably add some error handling in case someone enters `watch notAResourceKind`

Reply:
Updated sample feature to grab a list of available resources to watch and use this list to validate user input.